### ISSUE TITLE
Glitnir has WORD as token type

### ISSUE THREAD

**Issue by busla** [vthorsteinsson/Reynir#57]
I guess this should be ENTITY, correct?

```json
      {
        "b": "EFET", 
        "c": "kk", 
        "f": "alm", 
        "k": "WORD", 
        "s": "glitnir", 
        "t": "no_et_ef_kk", 
        "x": "Glitnis"
      }
```

**Comment by busla** [comment]
Text POST-ed to greynir API:


> Áherslan var lögð á óeðlilegar tilfærslur eigna í aðdraganda inngrips ríkisins í Glitni. Í þeirri vinnu voru tvær mismunandi dagsetningar sérstaklega notaðar til viðmiðunar. Önnur var 25. september 2008, en þann dag fór Þorsteinn Már Baldvinsson, þáverandi stjórnarformaður Glitnis, á fund Davíðs Oddssonar, þáverandi seðlabankastjóra til að lýsa fyrir honum áhyggjum sínum af þeirri stöðu sem upp var komin í rekstri Glitnis.

**Comment by vthorsteinsson** [comment]
Yes, "Glitnis" in this case should probably be seen as an ENTITY/sérnafn. The slight complication is that "glitnir" with a lower case "g" is allowed as a noun in BÍN, so Greynir is falling back on that. But it probably shouldn't. I will look into it.
